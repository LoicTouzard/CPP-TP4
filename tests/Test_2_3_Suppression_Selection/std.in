C c1 0 0 1
R rect -1 1 1 -1
S s -3 3 3 -3
DELETE s
LIST
EXIT
