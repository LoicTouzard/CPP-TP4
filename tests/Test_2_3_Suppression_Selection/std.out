OK
#New object :c1
OK
#New object :rect
OK
#New object : s
OK
#The list is empty
